demo-simple/src/main/AndroidManifest.xml

```diff
@@ -3,13 +3,13 @@
   package="com.novoda.downloadmanager.demo">
 
   <application
-    android:allowBackup="true"
+    android:name="com.novoda.downloadmanager.demo.DemoApplication"
+    android:allowBackup="false"
```

Comment:
@zegnus not sure why we made this `allowBackup` for the demo, put to false as it seems unnecessary.